### ISSUE TITLE
Error in the snd_cs46xx driver

### ISSUE THREAD

**Issue by karstengit** [opensrc/alsa#17]
Found in the syslog

```
Jun 17 20:25:34 PC8 pulseaudio[2244]: ALSA weckte uns auf, um neue Daten auf das Gerät zu schreiben, doch es gab nichts zum Schreiben!
Jun 17 20:25:34 PC8 pulseaudio[2244]: Dies ist höchstwahrscheinlich ein Fehler im ALSA-Treiber 'snd_cs46xx'. Bitte melden Sie diesen Fehler den ALSA-Entwicklern.
Jun 17 20:25:34 PC8 pulseaudio[2244]: Wir wurden durch das POLLOUT-Set geweckt, allerdings lieferte ein anschliessender snd_pcm_avail() den Wert 0 oder einen anderen Wert < min_avail.
```

translated:

```
June 17 20:25:34 PC8 pulseaudio [2244]: ALSA woke us up to write new data to the device, but there was nothing to write!
June 17 20:25:34 PC8 pulseaudio [2244]: This is most likely an error in the ALSA driver 'snd_cs46xx'. Please report this error to the ALSA developers.
June 17 20:25:34 PC8 pulseaudio [2244]: We were awakened by the POLLOUT set, however, yielded a subsequent snd_pcm_avail () 0 or another value <min_avail.
```

**Comment by markc** [comment]
This is really hard to diagnose and may be very specific to your particular hardware so I think the best thing you could do is to download this script and run it, which will offer to upload the output to alsa-project.org, and sign up to the alsa-user mailing-list with a description of your problem with error messages and a link to the uploaded info.
 
http://www.alsa-project.org/alsa-info.sh

http://lists.sourceforge.net/lists/listinfo/alsa-user

**Comment by karstengit** [comment]
I run the script and uploaded the contents.
I attach it here additional.

[alsa-info.txt.KLWKGxVZvS.zip](https://github.com/opensrc/alsa/files/322213/alsa-info.txt.KLWKGxVZvS.zip)

Why i must use the mailing list?
It is a better place to handle bug reports here?

**Comment by markc** [comment]
Not really as this is not an official ALSA project site, just a legacy documentation site. The developers and most active users will be reachable via either the alsa-user or alsa-devel mailing-lists.

Just a long shot suggestion, make absolutely sure your desktop user belongs to the `audio` group. Just type "groups" into a terminal app and, if not, then `gpasswd -a YOUR_USER audio` and log back in.

**Comment by karstengit** [comment]
Yes all the users belong to the audio group:
karsten,multimedia,timidity,pulse

This should not be the problem.

**Comment by markc** [comment]
Yes, sign up to the alsa-users list and submit your translated errors in the first issue, any other info like what apps you were using at the time, and a link to your alsa-info.sh script output so you don't have to go into the tedious details about your alsa, kernel and hardware versions.

**Comment by karstengit** [comment]
O.K. done

**Comment by markc** [comment]
Great, good luck. It's the best place to get some ALSA help so it won't be a waste of time now or in the future. Feel free to "Close and comment" this ticket with the (hopefully successful) results of your research into your problem.

**Comment by karstengit** [comment]
I asked for my problems at the ALSA mailing lists - without an answer! :-(
Here is no solution too :-(
